com.adamcake.Bolt.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.adamcake.Bolt
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: bolt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+modules:
+  - name: python-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install . --prefix=${FLATPAK_DEST}
+    build-options:
+      build-args:
+        - --share=network
+    sources:
+      - type: archive
+        url: https://github.com/python-pillow/Pillow/archive/refs/tags/10.0.0.tar.gz
+        sha256: 535d17e830427bec163027114ded1def9ab0350c99bf1d8cb10535032967f3a5
+  - name: bolt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBOLT_BINDIR=bin
+    sources:
+      - type: archive
+        dest: cef/dist
+        url: https://adamcake.com/cef/cef-114.0.5735.201-linux64-minimal-ungoogled-flatpak.tar.gz
```

Comment:
so how does this work on aarch64?

Reply:
Ah, good question. This will not build on anything other than x86_64 at the moment. Is 6d50f5c sufficient to fix this for now? The source is a custom build of libcef with ungoogled-chromium and org.chromium.Chromium patchsets applied to it. I could set up build rules for this instead of shipping binaries, but it would be no small undertaking for me and the builds would take 10-12 hours, which I don't know if flathub would be okay with.